webconsole/src/main/resources/res/ui/config.js

```diff
@@ -348,6 +349,25 @@ function printConfigurationInfo( /* Element */ parent, obj )
 
 
 var spanCounter = 0;
+/* Element */ function createAddButton(prop) {
+    spanCounter++;
+    var newId = prop + spanCounter;
+
+    var addButton = createElement("input", null,
```

Comment:
maybe one could reuse the same code here and in createSpan(...) to reduce duplication.